### ISSUE TITLE
Couln't get it to run

### ISSUE THREAD

**Issue by TBubba** [07box/07box#1]
I know this is probably dead, but I would really appreciate something like this.

Tried to run the newest commit, but it crashed because one of a "BadImageFormatException" at line 33 in "Program.cs". I couldn't even find the code that actually loaded the dll, since the debugger didn't tell me.

Error message:
http://pastebin.com/5P1P8bAE

**Comment by 07box** [comment]
It's been 2 years, I don't think anything works anymore. Sorry. You do know that Orion client has the resizable functionality now?

**Comment by TBubba** [comment]
Yeah, I recently found out :]. Also, Jagex stated on Twitter (yesterday I think) that they would implement resolution settings this summer or something along those lines.
